copter/source/docs/traditional-helicopter-internal-rsc-governor.rst

```diff
@@ -0,0 +1,46 @@
+=====================
+
+ArduPilot provides an internal RSC governor (:ref:`H_RSC_MODE<H_RSC_MODE>` =4) for those applications which do not have one externally to assure constant rotor head speed. This mode requires the setup of the ``H_RSC_THRCRV_x`` and ``H_RSC_GOV_x`` parameters and the use of an :ref:`RPM sensor<common-rpm>` for the rotor head speed.
+
+The new governor settings are:
+
+- :ref:`H_RSC_GOV_DISGAG<H_RSC_GOV_DISGAG>`:The disengage point: recommended setting 2% above your flight idle power setting (first point on the throttle curve). As shown in the below example, the governor disengages at 20% throttle.
+- :ref:`H_RSC_GOV_DROOP<H_RSC_GOV_DROOP>`:Droop Response setting: 0-100%. This sets how aggressive the governor is. A droop response setting too high in relation to how your prime mover reacts to throttle can cause the governor to surge or over-react to load change. Around 85% should work good for most piston engines. A little higher for turbines. A lot lower for electrics.
+- :ref:`H_RSC_GOV_RANGE<H_RSC_GOV_RANGE>` :RPM range above or below :ref:`H_RSC_GOV_SETPNT<H_RSC_GOV_SETPNT>` setting where governor is operational. If speed sensor fails, or rpm falls outside of this range, the governor will disengage and return to throttle curve. Recommended range is 100 rpm.
+- :ref:`H_RSC_GOV_SETPNT<H_RSC_GOV_SETPNT>`:RPM setpoint - set to desired headspeed in RPM
+- :ref:`H_RSC_GOV_TCGAIN<H_RSC_GOV_TCGAIN>`:Throttle Curve Gain - set to provide feedforward from the throttle curve for sudden change in load. If it over-runs on a sudden punchout, reduce the TC gain, etc… Can be set from 50-100%
+
+
+Setup
+=====
+
+You must first set your throttle curve and properly tune it. If the sensor fails, control fails over to the throttle curve. Without a properly tuned curve, your helicopter will crash. For UAV operation it is recommended to set your throttle curve with a flight idle position (the throttle curve 0 point). This allows you to idle your ICE or Turbine engine down by pulling the collective to the bottom. Dis-engaging the motor interlock in ArduPilot will also set the RSC output for motor throttle to :ref:`H_RSC_IDLE<H_RSC_IDLE>`.
+
+When tuning your throttle curve, tune it aggressively so it runs higher rpm than you normally fly at. This ensures adequate power available if the sensor fails. You don’t have to have a perfectly tuned throttle curve, but it should be “close enough” to fly the heli with no surprises. You can tune your curve with :ref:`H_RSC_MODE<H_RSC_MODE>` =3. Or tune it with :ref:`H_RSC_MODE<H_RSC_MODE>` =4,but with the :ref:`H_RSC_GOV_TCGAIN<H_RSC_GOV_TCGAIN>` set to 100 and :ref:`H_RSC_GOV_DROOP<H_RSC_GOV_DROOP>` set to 0. Either way works.
+
+You can refer to the :ref:`ArduPilot Helicopter Ground School videos<traditional-helicopter-configuration>` if you want to take a refresher on setting throttle curves. Video 4 covers throttle curves, piston and turbine engines:
+
+Logging Governor Performance
```

Comment:
I think this is not the right heading, at least no for anything beyond the first paragraph.  Certainly we can add a section for Governor logging but everything below the first paragraph could probably be titled "Helpful Tips" or "Tuning Tips".  I might need to look at this closer and adjust.

Reply:
not sure what you mean...lets chat...

Reply:
will fix as discussed